test/tests/providers/global/mimir.rules.test.yml

```diff
@@ -86,3 +82,28 @@ tests:
               description: "Mimir ruler is failing to process PrometheusRules."
       - alertname: MimirRulerEventsFailed
         eval_time: 160m
+  - interval: 1m
+    input_series:
+      - series: 'kube_pod_container_status_restarts_total{cluster_type="management_cluster", namespace="mimir"}'
```

Comment:
```suggestion
      - series: 'kube_pod_container_status_restarts_total{cluster_type="management_cluster", namespace="mimir", container!="prometheus"}'
```

Reply:
I think we need to set a container that is not prometheus 😅

Reply:
But then we probably need another input with container=prometheus

Reply:
It's fixed in later version :)